### ISSUE TITLE
The error is

### ISSUE THREAD

**Issue by shriyashkorade** [Sathvik149/Pharmacy-Drug-Management-System#3]
![Screenshot 2024-02-20 234135](https://github.com/Znsored/Pharmacy-Drug-Management-System/assets/109428326/f81c9d88-dcd4-42e5-9bad-c61bdfc5b023)

**Comment by Znsored** [comment]
This issue is related to your software installation and IDE setup. Search for tutorials on how to setup all the listed software and libraries.

**Comment by shriyashkorade** [comment]
> This issue is related to your software installation and IDE setup. Search for tutorials on how to setup all the listed software and libraries.

Please, can you help me with this issue.?

**Comment by Znsored** [comment]
no, this project is 5 years old. I do not remember anything apart from the code.
